src/apiCalls.js

```diff
@@ -28,7 +28,7 @@ const postComment = (newComment) => {
   })
   .then((resp) => {
     if (!resp.ok) {
-      throw new Error(resp.statusCode)
+      throw new Error(`${resp.statusCode}: Oops, we encountered an issue submitting your comment. Please try again.`)
```

Comment:
all above error messages are looking like client side facing, it's fine as long as you understand users are not going to see these messages. They would only show up in devtools.

Reply:
You're right! I will make those changes on the calls.